shop/shop-repository/src/main/java/shop/woowasap/shop/repository/MockProductRepository.java

```diff
@@ -70,7 +89,8 @@ public ProductsPaginationResponse findAllValidWithPagination(int page, int size)
             .startTime(INFINITE_START_TIME.toInstant(ZoneOffset.of(OFFSET_ID)))
             .endTime(INFINITE_END_TIME.toInstant(ZoneOffset.of(OFFSET_ID)));
 
-        List<Product> products = List.of(productBuilder.id(1L).build(), productBuilder.id(2L).build());
+        List<Product> products = List.of(productBuilder.id(1L).build(),
```

Comment:
여기 컨벤션 final이 빠진거 같아요~
어차피 곧 삭제될 클래스인거 같아서 그냥 넘기셔도 될거같습니다~

(혹시, 수정하고 싶으실까봐! 알림용으로 달아드려요)

```suggestion
        final List<Product> products = List.of(productBuilder.id(1L).build(),
```